Src/WitsmlExplorer.Api/Workers/DownloadLogDataWorker.cs

```diff
@@ -114,4 +149,361 @@ private string GetReportBody(ICollection<Dictionary<string, LogDataValue>> repor
+        WriteWellParameter(writer, "SAPPC", "", "", "APPLIED CORRECTIONS", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SDATE", "", $"{DateTime.Now.ToShortDateString()} {DateTime.Now.ToShortTimeString()}", "DATE", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SCLAB", "", "", "County label", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SSLAB", "", "", "State/Province label", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SPROV", "", "", "State or Province", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SCTRY", "", well.Country, "COUNTRY", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SCONT_REGION", "", "", "Continent Region", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SSECT", "", "", "Section", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "STOWN", "", "", "Township", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SRANGE", "", "", "Range", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SAPI", "", "", "API Number", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SUWI", "", "", "UNIQUE WELL IDENTIFIER", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SLUL", "", "", "Logging Unit Location", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SLUN", "", "", "Logging Unit Number", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SLOC", "", "", "Field Location", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SFL1", "", "", "Field Location line 1", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SFL2", "", "", "Field Location line 2", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SLATI", "deg", "", "Latitude", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SLONG", "deg", "", "Local Permanent Datum", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SPDAT", "", "", "Geodetic Datum", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SGDAT", "", "", "Geodetic Datum", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SLMF", "", "", "Logging Measured From", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SAPD", "", "", "Elevation of Depth Reference (LMF) above Permanent Datum", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SEPD", "", "", "Elevation of Permanent Datum (PDAT) above Mean Sea Level", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SEKD", "", "", "Elevation of Kelly Bushing above Permanent Datum", maxColumnLength, maxDataLenght);
+        WriteWellParameter(writer, "SEDF", "", "", "Elevation of Drill Floor above Permanent Datum", maxColumnLength, maxDataLenght);
```

Comment:
Most of these are not specified. I think we should remove them. We can rather add values later if requested.

We could try to support every value as specified in the standard, but then we would have to use the WitsmlWell and WitsmlLog models (which we don't get when we use the well and log services, but we could still easily get these). The required values are specified here: https://www.cwls.org/wp-content/uploads/2017/02/Las2_Update_Feb2017.pdf